### PR TITLE
Feature/cronjob trigger process

### PR DESCRIPTION
**Pull request by kafkaphoenix**
## Description

New process cronjob trigger that given a cron expression and a message will send that message through sdk messaging with a time bases depending on the cron

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Other changes (ci configuration, documentation or any other kind of changes)

## Checklist:

- [ ] I have created tests for my code changes, and the tests are passing.
- [ ] I have executed the pre-commit hooks locally.
- [ ] My change requires a change to the documentation (create a new issue if the documentation has not been updated).
- [ ] I have updated the documentation accordingly.

authors: @alexrodfe @kafkaphoenix

### CODE REVIEW

File: cronjob-trigger/README.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+The cronjob trigger is a predefined KAI process that will trigger an action in a predefined time.  
+
+## How to setup
+
+The trigger requires adding two configuration options:
+- A predefined cron time with the structure `cron: * * * * * *` which equals to seconds | minutes | hours | days | months | years
+- The message to be sent
+
+### Input
```
Comment (alexrodfe) [inline cronjob-trigger/README.md:11]: Instead of input I believe this should be "config" then tell the "keys" the .yaml is waiting for

**Review by alexrodfe — APPROVED** [review]

File: cronjob-trigger/README.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+The configuration should be defined inside the `centralized configuration scope`, for example:
+
+```
+centralized_configuration:
+  process:
+    bucket: process
+    config:
+      cron: '30 * * * * *'
+      message: 'Hello world'
+```
```
Comment (dfinteligenz) [inline cronjob-trigger/README.md:22]:
```suggestion
### Config 

The configuration should be defined inside the `centralized configuration scope`, for example:

| Key            | Optional  | Type | Value                                                                                         |
|----------------|-----------|------|-----------------------------------------------------------------------------------------------|
| cron | no        | str  | Cron expression     |
| message | yes        | str  | Message to be sent in the generated events      |
```

File: cronjob-trigger/README.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+
+## Output
+
+It triggers an event of sending a message through the module messaging in the sdk in the predefined interval of time.
+
```
Comment (dfinteligenz) [inline cronjob-trigger/README.md:27]: Add the response event structure as a table with the format:

```
| Key       | Type | Value                                                                  |
|-----------|------|------------------------------------------------------------------------|
| requestID | str  | A string uuid                                                          |
| message  | str  | The defined message (if any(                             |
| time     | str  | The timestamp of the generated event                                |
```

Comment (dfinteligenz) [reply cronjob-trigger/README.md:27]: Also, add a real JSON example of the event below.

File: cronjob-trigger/app.yaml [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+  base_path: '/Some/Path'
+
+nats:
+  url: 'nats://localhost:4222'
+  stream: cronjob-trigger
+  output: cronjob-trigger-output
+
+centralized_configuration:
+  product:
+    bucket: product
```
Comment (dfinteligenz) [inline cronjob-trigger/app.yaml:15]:
```suggestion
centralized_configuration:
  global:
    bucket: global
  product:
    bucket: product
```

File: github-webhook-trigger/README.md
Comment (dfinteligenz) [inline github-webhook-trigger/README.md:1]: This has been already changed in the PR https://github.com/konstellation-io/kai-processes/pull/14, please revert this changes @kafkaphoenix

**Review by dfinteligenz — CHANGES_REQUESTED** [review]

**Review by alexrodfe — APPROVED** [review]

File: github-webhook-trigger/README.md [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+- release
+- workflow_dispatch
+- workflow_run
+
+## How to setup
+
+The trigger requires adding two configuration options to the process-scoped configuration.
+One being the events the webhook will listen to (_webhook_events_), the other the github secret needed to interact with the github repo (_github_secret_).
+
+### Input
```
Comment (dfinteligenz) [inline github-webhook-trigger/README.md:17]:
```suggestion
## Input 
```

File: cronjob-trigger/README.md [hunk 6]
```diff
@@ -0,0 +1,50 @@
...
+The cronjob trigger is a predefined KAI process that will trigger an action in a predefined time.  
+
+## How to setup
+
+The trigger requires adding two configuration options:
+- A predefined cron time with the structure `cron: * * * * * *` which equals to seconds | minutes | hours | days | months | years
+- The message to be sent
+
+### Config
```
Comment (dfinteligenz) [inline cronjob-trigger/README.md:11]:
```suggestion
## Input 
```

File: cronjob-trigger/README.md [hunk 7]
```diff
@@ -0,0 +1,50 @@
...
+### Config 
+
+The configuration should be defined inside the `centralized configuration scope`:
+
+| Key            | Optional  | Type | Value                                                                                         |
+|----------------|-----------|------|-----------------------------------------------------------------------------------------------|
+| cron | no        | str  | Cron expression     |
+| message | yes        | str  | Message to be sent in the generated events      |
+
+#### Example
```
Comment (dfinteligenz) [inline cronjob-trigger/README.md:20]:
```suggestion
### Example
```

**Review by dfinteligenz — CHANGES_REQUESTED** [review]
Check the comments above

**Review by dfinteligenz — APPROVED** [review]

**Pull request merged**